notebooks/evaluation/README.md

```diff
@@ -18,15 +18,16 @@ With such a vast hyperparameter space, the first principle of our evaluation is
 > [!NOTE]  
 > **Data is immutable.** Analysis is done on static, versioned datasets so our insights share a common subject.
 
-We therefore conceptualise evaluation as containing two roles:
+We therefore conceptualise evaluation as containing three roles:
 
 * I want to **create a new versioned dataset** for my colleagues to study
+* I want to **baseline a new versioned dataset** to create metrics to improve on
```

Comment:
Do we want to add something to the README to say that people can contact the Redbox team to get access to a pre-existing versioned dataset?